src/lib/core/components/visualizations/implementations/ScatterplotVisualization.tsx

```diff
@@ -1025,6 +1049,31 @@ function ScatterplotViz(props: VisualizationProps) {
             ) : undefined,
         ]}
       />
+
+      {/* show message if no smoothed mean exists */}
+      {!data.pending &&
+        vizConfig.valueSpecConfig === 'Smoothed mean with raw' &&
+        dataWithoutSmoothedMean != null &&
+        dataWithoutSmoothedMean?.length > 0 && (
+          <div
```

Comment:
Can we use Jeremy's banner component from CoreUI here?

Reply:
I didn't carefully check the CoreUI's banner before so will see whether it fits here.